partner_academic_title/data/partner_academic_title_data.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2015-2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <data noupdate="1">
```

Comment:
data element is no longer needed. noupdate attribute can be added to odoo element. Please remove data and decrease indents of records.